BurdaMagazinOrg/infinite-theme
# INREL-3764 new article trigger

## [INREL-3764](https://jira.burda.com/browse/INREL-3764) 
- new article_footer_blocks_common in common template. So we can add dynamically the next-article-info-layer for specific platform (InStyle next article scroll logic when you see the preloader

## How to test:
Scroll fast to the next article. Now you see the preloader. After the preloading is done it's scrolling automatically to the next article.

## Checklist:
